src/player.js

```diff
@@ -0,0 +1,57 @@
+class Player {
+  constructor(col, row, img) {
+    this.col = col;
+    this.row = row;
+    this.width = PLAYER_WIDTH;
+    this.height = PLAYER_HEIGHT;
+    this.img = img;
+  }
+
+  preLoad() {
+    this.img = loadImage("./assets/character-down.png");
+  }
+
+  drawPlayer() {
+    image(this.img, this.row, this.col, this.width, this.height);
+  }
+
+  moveUp() {
+    if (keyDown(ARROW_UP)) {
+      this.row -= PLAYER_MOVEMENT;
+
+      if (this.row <= this.height) {
+        this.row = HEIGHT;
```

Comment:
This logic is not quite there yet! I'll leave it to you to figure out why, but think about what the value of `HEIGHT` is 🤔